docs/source/TaskEngineWorkflows.md

```diff
@@ -348,6 +362,43 @@ This workflow allows you to create an MP4 from a VOD asset
 }
 ```
 
+## Build_thumbnails
+
+This workflow allows you to generate thumbnail assets which can then be used for video timeline previews.
+
+### Build_thumbnails: Parameters
+
+| Parameter Name    | Required |  Description | Default |
+| ----------------- | -------- | ------------ | ------- |
+| workflow          |Yes| Specify 'build_thumbnails'. ||
+| content_id        |Yes| Unique identifier of the content. This is usually a key that allows identification of the content in the client’s system.||
+| source            |Yes| URL of the HLS source from which to create assets (isml sources must be in a state of `stopped`). ||
```

Comment:
I would say live streams instead of isml soruces. You can include .isml in brackets next to it.